pkg/util/azure/test/azure_metadata.json

```diff
@@ -0,0 +1,82 @@
+{
+  "azEnvironment": "AzurePublicCloud",
+  "customData": "",
+  "location": "centralus",
+  "name": "negasonic",
+  "offer": "lampstack",
+  "osType": "Linux",
+  "placementGroupId": "",
+  "plan": {
+      "name": "5-6",
+      "product": "lampstack",
+      "publisher": "bitnami"
+  },
+  "platformFaultDomain": "0",
+  "platformUpdateDomain": "0",
+  "provider": "Microsoft.Compute",
+  "publicKeys": [],
+  "publisher": "bitnami",
+  "resourceGroupName": "myrg",
+  "resourceId": "/subscriptions/xxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx/resourceGroups/myrg/providers/Microsoft.Compute/virtualMachines/negasonic",
+  "sku": "5-6",
+  "storageProfile": {
+      "dataDisks": [
+        {
+          "caching": "None",
+          "createOption": "Empty",
+          "diskSizeGB": "1024",
+          "image": {
+            "uri": ""
+          },
+          "lun": "0",
+          "managedDisk": {
+            "id": "/subscriptions/xxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx/resourceGroups/macikgo-test-may-23/providers/Microsoft.Compute/disks/exampledatadiskname",
+            "storageAccountType": "Standard_LRS"
+          },
+          "name": "exampledatadiskname",
+          "vhd": {
+            "uri": ""
+          },
+          "writeAcceleratorEnabled": "false"
+        }
+      ],
+      "imageReference": {
+        "id": "",
+        "offer": "UbuntuServer",
+        "publisher": "Canonical",
+        "sku": "16.04.0-LTS",
+        "version": "latest"
+      },
+      "osDisk": {
+        "caching": "ReadWrite",
+        "createOption": "FromImage",
+        "diskSizeGB": "30",
+        "diffDiskSettings": {
+          "option": "Local"
+        },
+        "encryptionSettings": {
+          "enabled": "false"
+        },
+        "image": {
+          "uri": ""
+        },
+        "managedDisk": {
+          "id": "/subscriptions/xxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx/resourceGroups/macikgo-test-may-23/providers/Microsoft.Compute/disks/exampleosdiskname",
+          "storageAccountType": "Standard_LRS"
+        },
+        "name": "exampleosdiskname",
+        "osType": "Linux",
+        "vhd": {
+          "uri": ""
+        },
+        "writeAcceleratorEnabled": "false"
+      }
+  },
+  "subscriptionId": "xxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx",
+  "tags": "Department:IT;Environment:Test;Role:WebRole",
```

Comment:
We need to also pull the actual tags set by users.

Reply:
I wasn't sure whether I should include this initially. Should there be an exclude filter similar to how the [GCE filter](https://github.com/DataDog/datadog-agent/blob/8086c3303dc785d560f1b73e031c4dc3a722ca0a/pkg/util/gce/gce_tags.go#L91-L100) works?

Reply:
Yes, might be a good idea